backend/pyproject.toml

```diff
@@ -10,6 +10,7 @@ readme = "README.md"
 python = "^3.10"
```

Comment:
On the Readme, it would be nice to document a really minimal quick set of instructions at least at the top level, e.g.:
```md
# playlist-manager

## Basic set up

Requirements:
* Python
* Poetry
* Node & npm

`bash install.sh`

Set up an app registered at <https://developer.spotify.com/dashboard>

Set a Redirect URL for `http://localhost:1234`.

Create a `.env` file in the `backend` folder based on `.env.template`, and set the appropriate missing variables using those from the Spotify dashboard.

## Running it

`bash run-dev.sh`
```

Reply:
Ah yeah, sorry this is all really useful and I should have remembered to update the README as I was implementing some of this!